test/browser/replaceNode.test.js

```diff
@@ -135,9 +135,10 @@ describe('replaceNode parameter in render()', () => {
 		const expectedB = '<div id="b">childB</div>';
 		const expectedC = '<div id="c">childC</div>';
 
-		render(<div id="a">childA</div>, scratch, childA);
-		render(<div id="b">childB</div>, scratch, childB);
-		render(<div id="c">childC</div>, scratch, childC);
+		// TODO: text nodes are somehow left out.
+		render(<div id="a">childA</div>, childA);
```

Comment:
The issue here is that `oldRoot` will be used for different `replaceNode` calls so it will effectively always overwrite previous results.

It should not preserve `oldRoot` for a different `replaceNode` argument